pkgs/by-name/ot/oterm/package.nix

```diff
@@ -9,36 +9,26 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "oterm";
-  version = "0.11.1";
+  version = "0.12.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "ggozad";
     repo = "oterm";
     tag = version;
-    hash = "sha256-b/+siNzmM6RUJ3jv/2dNJJFueejChKde0D5r8J0lTqM=";
+    hash = "sha256-6y73Lh3cV/fnvpZWzfyD4CletC4UV2zl+I7l88BYPIk=";
   };
 
-  pythonRelaxDeps = [
-    "aiosql"
-    "aiosqlite"
-    "httpx"
-    "ollama"
-    "packaging"
-    "pillow"
-    "pydantic"
-    "textual"
-    "typer"
-  ];
+  pythonRelaxDeps = true;
```

Comment:
Thanks for updating this package as well. However I'd prefer to keep explicitly listing the relaxed deps. Thanks to this, my change to `mcp` broke the build of `oterm`, instead of building fine and releasing a broken version in terms of its mcp functionality. So we caught the problem early and I was able to request upstream adaptation of `oterm`.

Reply:
Ok fair enough!